source/2019-08-02-the-ember-times-issue-109.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: The Ember Times - Issue No. 109
-author: Chris Ng, the crowd
+author: Chris Ng, Amy Lam, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/08/02-the-ember-times-issue-109.html"
 responsive: true
 ---
 
-<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+🙌 Emberistas! 
 Read the newly published Ember 2019-2020 Roadmap RFC 🐹,
+GraphQL and Ember 👨‍💻,
```

Comment:
Maybe "Meet Ember developers at This.JavaScript: GraphQL" instead?

Reply:
changed to `tune in to learn about GraphQL and Ember 👨‍💻` cause it's an online livestream or watch the recording after... open to suggestions though